.github/workflows/docs/publish-docs.yml

```
@@ -0,0 +1,67 @@
+    push:
+        branches:
+            - '**'
+        paths:
+            - 'docs/**'
+    pull_request:
+        branches:
+            - '**'
+        paths:
+            - 'docs/**'
+jobs:
+    deploy-docs:
```

actionlint (1.7.4)

job "deploy-docs" needs job "test-docs" which does not exist in this workflow

(job-needs)